install.sh

```diff
@@ -463,7 +469,11 @@ killtree() {
 killtree $(lsof | sed -e 's/^[^0-9]*//g; s/  */\t/g' | grep -w 'k3s/data/[^/]*/bin/containerd-shim' | cut -f1 | sort -n -u)
 
 do_unmount() {
-    MOUNTS=$(cat /proc/self/mounts | awk '{print $2}' | grep "^$1" | sort -r)
+    MOUNTS=
+    while read ignore mount ignore; do
+        MOUNTS="$mount\n$MOUNTS"
+    done </proc/self/mounts
+    MOUNTS=$(printf $MOUNTS | sort -r -)
```

Comment:
It appears that this change is missing the `grep "^$1"` functionality, which causes everything in `/proc/self/mounts` to be unmounted. 
(╯°□°)╯︵ sʇunoɯ/ɟlǝs/ɔoɹd/